docker-compose.yml

```
@@ -3,12 +3,13 @@ version: "3.8"
 services:
   nginx:
     restart: unless-stopped
-    build: ./nginx
+    build:
+      target: prod
+      context: ./web
     image: boilerplate.azurecr.io/nginx:latest

```

```suggestion
    image: ghcr.io/equinor/template-fastapi-react/nginx:latest

```
I'd suggest that we stick to a single source for our Docker images, unless there's any particular reason for this.

---

Has the name changed since I started working on this?
(since I didn't change the name)

---

Ah, you're right. Unrelated to this PR; just caught my eye. Feel free to skip for this PR; I'll make an issue for a separate PR.